src/MongoDB.Driver.Core/Core/Connections/IConnection.cs

```diff
@@ -81,6 +78,12 @@ public interface IConnection : IDisposable
         /// </value>
         ConnectionSettings Settings { get; }
 
+        /// <summary>
+        /// Add additional read timeout value.
+        /// </summary>
+        /// <param name="timeout">The timeout.</param>
+        void AddAdditionalReadTimeout(TimeSpan timeout);
```

Comment:
This doesn't match what's on your csharp2964 branch.

Can you please check what's going on?

I don't think there's any point in reviewing any further if this pull request doesn't match your branch.